### DISCUSSION TITLE
Is it possible to get release notes from azure git tags not last successful build/release?

### DISCUSSION THREAD

**Discussion by mahdiahmadirad** [rfennell/AzurePipelines#1554] [Q&A]
for our major releases we set a tag in repos. so we want to collect release notes from workItems associated between current and previous tags. is it possible to handle with your incredible extension?

**Comment by rfennell** [comment]
The task will use Azure DevOps pipeline tags (the `tags` parameter), but does not support Git Tags. This is because it queries the Azure DevOps pipeline API not the Repo API

So, the answer is no it can't use Git tags without a major change to the core logic
